sdk-ts/src/utils/AccountParser.ts

```diff
@@ -210,34 +210,53 @@ export class AccountParser {
     }
 
     // reward
+
     const reward_vesting: Decimal[] = [];
     const reward_base = uai_base + uai_size * 16;
-    for(let i = 0; i < 7; i++) {
+    for (let i = 0; i < 4; i++) {
       const r_offset = reward_base + i * 8;
       reward_vesting.push(AccountParser.parseFloat64(data.buffer, r_offset));
     }
     const reward = {
       vesting: reward_vesting,
+      prev_week_apt: AccountParser.parseFloat64(
+        data.buffer,
+        reward_base + 8 * 4
```

Comment:
Note that 8 * 5 and 8 * 6 are skipped because these two 8 bytes are just paddings.